python/ray/runtime_context.py

```diff
@@ -381,6 +381,17 @@ def _get_actor_call_stats(self):
         worker.check_connected()
         return worker.core_worker.get_actor_call_stats()
 
+    def get_resource_ids(self) -> Dict[str, List[Tuple[int, float]]]:
```

Comment:
nit: 
to compatible with `ray. get_gpu_ids`,  we should return a Dict[Str, List[int]]

Reply:
Now I see an existing deprecated `get_resource_ids` (in worker) as well as `get_available_resources` (in runtime-context).
Likely we can name it as:
1. get_gpu_and_accelerator_ids => returns GPU/NeuronCore/...
2. get_accelerator_ids => returns NeuronCore/...

Reply:
Also, I see one inconsistency with get_gpu_ids today

If no previous CUDA configuration, we return List[int]
If CUDA configuration, we return List[str] as the CUDA configuration can have UUID

Is making get_gpu_ids to return List[str] all the time a breaking change for existing workloads?

Reply:
Per offline discussion, we decided not to touch `get_gpu_ids` and `runtime_ctx.get_resource_ids` will always return List[str]